src/components/ciConfig/CIConfig.scss

```diff
@@ -67,20 +83,103 @@
             }
         }
     }
+
     .popup-button {
         .select-button {
             height: 40px;
             padding-left: 10px;
         }
     }
+
     .target-platform__select__multi-value__label {
         font-size: 12px;
         padding: 2px;
         padding-left: 6px;
     }
 }
+
 .docker__dropdown {
     margin: auto;
     margin-right: 0;
     margin-top: 0;
 }
+
+.form__app-compose {
+    &.config-override-view {
+        width: 100%;
```

Comment:
Why aren't using short formulae classes here

Reply:
For using the formulae classed, I have to add additional checks so added the single check at the parent level and overridden the properties of the children wherever required in css.